README.md

```diff
@@ -44,6 +44,8 @@ More information about running `docker-compose` can be found [here.](https://doc
 
 [Note : Cloner clones the whole website, to restrict to a desired depth of cloning add `--max-depth` parameter]
 
+In case of an error while running `docker-compose up`, check the port 8080, 8090, 80 by running `sudo netstat -tulnp`, if they are active kill the process by sudo kill -9 <PID>  
```

Comment:
1. Snare docker runs on the port 80, so mentioning the 8080, 8090 is redundant
2. Killing the process might not be appropriate way for particular person, I would not recommend to put in in README
3. Please, only mention default port and that they might want to change it accordingly

Reply:
I see, thank you for the review. Please check new commit to change default port of 80 in Dockerfile and docker-compose.yml